### ISSUE TITLE
Make blocks more configurable

### ISSUE THREAD

**Issue by salashnik** [NEXSTEP/redmine_blocks_layout#1]
like https://github.com/finnlabs/chiliproject_my_project_page

**Comment by dexterns88** [comment]
yep , can you make option for setup where to show block on homepage.
Left or right column for beginning will be nice.

P.S. THX plugin work nice on verion 2.2

**Comment by cforce** [comment]
Boxes could also be displayed at one place in a tabbed style. See
http://www.redmine.org/attachments/download/8795/Additional_ProjectOverview_02.png for example.

The related issue in redmine is here http://www.redmine.org/issues/12789
